### ISSUE TITLE
Add satelite mosaic of planets?

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/gmtserver-admin#194]
I wonder if we should add the satelite mosaic of others celestial bodies (just like the NASA blue/black marble). Let me know what you think. I could do it if we agree. 

Here there are some links that could be used: 

Moon:
https://astrogeology.usgs.gov/search/map/Moon/Kaguya/TC/Ortho/v02/Lunar_Kaguya_TC_Ortho_Global_64ppd_v02
https://astrogeology.usgs.gov/maps/moon-kaguya-tc-global-mosaic

Mars|
https://murray-lab.caltech.edu/CTX/

**Comment by joa-quim** [comment]
Honestly I think maybe not. Need to not forget that we are creating (well, extending) a maintenance burden.

**Comment by seisman** [comment]
I would say yes because research about Mars are very hot recently.

**Comment by PaulWessel** [comment]
Since this should hopefully be automated via recipes etc it may not be that a much maintenance, but we clearly need to make this a one-pus-button operation - right now there are too many steps.

**Comment by joa-quim** [comment]
I was thinking more to when we'll need to relocate the data in current gmtserver.

**Comment by Esteban82** [comment]
I agree with Joaquim that you have to be careful with adding more data because later you will have to do maintenance.

But, in this case, I think that the biggest effort will be only at the beginning. I don't think it will involve much work once the data is on the server. At least this seems to me based on what happens with the blue/black marble data.

**Comment by joa-quim** [comment]
> data is on the server

I still think you are failing to see my point. And what about we'll need to find a new server?

**Comment by Esteban82** [comment]
Ok, I see your point now.

But I think we could have that issue anyway. And the main issue would be the earth dems which are very big while the moon mosaic is 506 MB. 

- srtm01s: 41GB 
-  srtm03s: 6.8 GB
- earth_gebco/si:2.9 GB each
- earth_relief and synbath: 2.9 GB each
- Mars relief:  3.1 GB 



https://astrogeology.usgs.gov/search/map/Moon/Kaguya/TC/Ortho/v02/Lunar_Kaguya_TC_Ortho_Global_64ppd_v02

**Comment by Esteban82** [comment]
Is it possible to move this to "Discussions". I think that it is better place to put this instead of here (with the other issues). 
The same could be done with the other issues with the "question" label.

**Comment by PaulWessel** [comment]
Maybe @seisman or @maxrjones knows how to do that?

**Comment by maxrjones** [comment]
Yes, I'll transfer this issue and for future reference:
## once per repository
in https://github.com/GenericMappingTools/gmtserver-admin/settings, enable discussions:
<img width="758" alt="image" src="https://github.com/GenericMappingTools/gmtserver-admin/assets/14077947/46a1801d-60cd-42e9-b0a6-f2c27b719c6c">
## for each issue
use the convert to discussion option added to the sidebar:
<img width="232" alt="image" src="https://github.com/GenericMappingTools/gmtserver-admin/assets/14077947/deb1d055-16da-4395-bab7-356b83fd001b">
